### ISSUE TITLE
.foo:has(+.bar) selector does not work

### ISSUE THREAD

**Issue by gajus** [fb55/css-select#111]
Subject:

```html
<div class="foo">a</div>
<div class="bar">b</div>

```

Query:

```css
.foo:has(+.bar)

```

Expected: to match `.foo` element.

Actual: no match.

**Comment by strarsis** [comment]
@gajus: https://jsfiddle.net/xzb5qao2/1/

In browser (Chrome) console:
````
Uncaught DOMException: Failed to execute 'querySelectorAll' on 'Document': '.foo:has(+.bar)' is not a valid selector.
````

**Comment by strarsis** [comment]
@gajus: How are you using `css-select`? Are you using it with `cheerio`?

**Comment by gajus** [comment]
> @gajus: How are you using css-select? Are you using it with cheerio (related #110)?

I am. I have forked Cheerio and updated css-select to the latest version.

**Comment by gajus** [comment]
@fb55 Is there a workaround that you would suggest?

**Comment by fb55** [comment]
Turns out we fixed only the general case for traversal-first selectors in #43. I've added a fix for this case in c817280.

Please note that the two elements in question still need to have a parent element, as that is how we access the next siblings.
